04-HTML5-APIs/Ex01/script.js

```diff
@@ -0,0 +1,28 @@
+document.addEventListener('load', function() {
+		window.indexedDB = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
+		if (!window.indexedDB) {
+	 	window.alert("Your browser doesn't support a stable version of IndexedDB. Such and such feature will not be available.");
+		}
+		let request = window.indexedDB.open('TextDB', 3);
+		let db;
+		request.onsuccess = function(event) {
+	 	db = event.target.result;
+		};
```

Comment:
IndexDB is fine, just be aware of the indentation. Try to make it more readable.